pkg/kubectl/cmd/rollout/rollout_pause.go

```diff
@@ -125,7 +119,7 @@ func (o *PauseConfig) CompletePause(f *cmdutil.Factory, cmd *cobra.Command, out
 	return nil
 }
 
-func (o PauseConfig) RunPause() error {
+func (o PauseConfig) RunPause() {
```

Comment:
I never said that checkErr should exist somewhere else other than the caller. You just need to stop appending in the caller and exit early.